sdk/storage/Azure.Storage.DataMovement/KnownIssues.md

```diff
@@ -0,0 +1,13 @@
+# Known Issues
+
+Azure.Storage.DataMovement is still in beta. Not all functionality is currently implemented, and there are known issues that may cause problems for your application. Significant known issues are documented below
+
+## Preserving properties and metadata on copy
+
+When copying data between blobs, blob properties and blob metadata are not preserved. There is currently no support for this feature; properties and metadata can only be defined by the caller to be applied uniformly to each destination blob.
+
+This limitation also exists for share files and directories.
+
+### Client-side encryption support
+
+**There is no support for copying client-side encrypted blobs.** Since client-side encryption is built on blob metadata, the necessary information to decrypt the blob will not be transferred in a service-to-service copy. Azure.Storage.Blobs will not be able to decrypt the copied blob. If that metadata is lost, your blob contents will be lost forever.
```

Comment:
One more Known Issue Amanada and I discovered the other day that is probably worth mentioning? Seems trivial to just fix but it's not unfortunately :)

"If you set `DataTransferOptions.InitialTransferSize` larger than 256 MiB for a Block Blob copy, the transfer may fail if the blob being copied is larger than 256 MiB."